packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title2/sizes.ts

```diff
@@ -0,0 +1,24 @@
+import { TailwindHelpers } from '../../../../types';
+
+/**
+ * Returns the `sizes` variants for component `title2`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function titleSizes({ theme }: TailwindHelpers) {
+  return {
+    sm: {
+      fontSize: theme('fontSize.md')
+    },
+    md: {
+      fontSize: theme('fontSize.2xl'),
+      lineHeight: theme('lineHeight.md')
+    },
+    lg: {
+      fontSize: theme('fontSize.2xl'),
```

Comment:
We can skip this since is the same as the default, right?

```suggestion

```

Reply:
should 4xl, it has 48px